web-client/src/components/validation/Predictor.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="space-y-8">
+    <!-- test the model -->
+    <ButtonCard
+      class="mx-auto mt-10 lg:w-1/2"
+      :button-placement="'center'"
+      @action="inferModel()"
+    >
+      <template #title>
+        Infer your model
```

Comment:
just a language thing: let's change to sth like `Make predictions using your model`. i would also change the code names (doing inference is fine, but infering the model in statistics means training the model, so this will be confusing later on)

Reply:
Okay, sure!